source/lib/JMSLab/tests/_test_helpers.py

```diff
@@ -180,6 +180,30 @@ def standard_test(test_object, builder,
     if system_mock:
         system_mock.assert_called_once()
         system_mock.reset_mock()
+        
+def handout_standard_test(test_object, builder,
```

Comment:
@ew487 can you remind me why we've implemented this in `_test_helpers` rather than in `test_build_lyx`?

Is this function used anywhere outside of `test_build_lyx`?

Reply:
@jmshapir You're totally right! It is there only because I copied and pasted it from `standard_test()`. I will move it to `test_build_lyx`.

Reply:
Fixed in [0e20ac3](https://github.com/JMSLab/Template/pull/62/commits/0e20ac3da126f2533ffdca8a9c5ffd53dd62ea0b)!